src/Sylius/Bundle/PayumBundle/composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "sylius/payum-bundle",
     "type": "symfony-bundle",
-    "description": "Payum integration for Symfony e-commerce applications.",
+    "description": "PaymentRequest integration for Symfony e-commerce applications.",
```

Comment:
This don't have to change until SyliusPayumBundle will be droped.

Reply:
It's strange because I don't remember changing it, apparently some regexp caught it 🤔